lua/darkrp_config/settings.lua

```diff
@@ -69,7 +69,7 @@ GM.Config.deadtalk                      = true
 GM.Config.deadvoice                     = true
 -- deathpov - Enable/disable whether people see their death in first person view.
 GM.Config.deathpov                      = false
--- decalcleaner - Enable/Disable clearing every player's decals.
+-- decalcleaner - Enable/Disable clearing ever players decals.
```

Comment:
This doesn't seem right. If this grammar error exists in DarkRP I think it's better to fix it there.

Reply:
This is now fixed in DarkRP.